snuba_sdk/expressions.py

```diff
@@ -17,6 +18,8 @@ def validate(self) -> None:
         raise NotImplementedError
 
 
+alias_re = re.compile(r"^[a-zA-Z](\w|\.|\[|\])*$")
```

Comment:
I've used the convention before that read only globals are in all caps. Is that not the convention here?